packages/extension/src/ui/screens/ConfirmScreen.tsx

```diff
@@ -79,27 +84,29 @@ export const ConfirmScreen: FC<ConfirmScreenProps> = ({
       <H2>{title}</H2>
 
       {children}
-
-      <StickyButtonGroupVertical
-        as="form"
-        onSubmit={(e: any) => {
-          e.preventDefault()
-          return onSubmit?.()
-        }}
-      >
-        {!singleButton && (
-          <Button onClick={onReject} type="button">
-            {rejectButtonText}
-          </Button>
-        )}
-        <Button
-          disabled={disableConfirm}
-          style={{ backgroundColor: confirmButtonBgColor }}
-          type="submit"
+      <StickyGroup>
+        {footerChildren}
+        <ButtonGroupVertical
+          as="form"
+          onSubmit={(e: any) => {
```

Comment:
use the correct type in place of `any`
```suggestion
          onSubmit={(e: FormEvent) => {
```